core/src/test/java/org/jdbi/v3/core/h2/TestH2SqlArrays.java

```diff
@@ -44,7 +44,8 @@ public class TestH2SqlArrays {
         h.useTransaction(th -> {
             th.execute("DROP TABLE IF EXISTS uuids");
             th.execute("CREATE TABLE uuids (u UUID ARRAY)");
-    }));
```

Comment:
this one I am meh on. We always had the closing curly and closing brackets on the same line for lambdas.

Reply:
```
    public static H2DatabaseExtension h2Extension = H2DatabaseExtension.instance().withInitializer(h ->
        h.useTransaction(th -> {
            th.execute("DROP TABLE IF EXISTS uuids");
            th.execute("CREATE TABLE uuids (u UUID ARRAY)");
        }));
```
Like this?
I changed it.